### ISSUE TITLE
Snapshot listing failed

### ISSUE THREAD

**Issue by nicedevil007** [jimsalterjrs/sanoid#482]
Hey guys,

I'm trying to sync my snapshot from my host to a backup server.
I'm using the command with this syntax:

`syncoid --no-sync-snap --sendoptions="--raw -c -L" nas/docker root@192.168.1.250:backup/docker`

The next message I get in cli is this one:

`Snapshot listing failed, trying fallback command at /user/sbin/syncoid line 1550`

I'm using the following version of sanoid/syncoid:

```
root@ubuntu:~# syncoid -v
/usr/sbin/syncoid version 2.0.3
(Getopt::Long::GetOptions version 2.50; Perl version 5.28.1)
```

The OS is ubuntu 19.10 on kernel 5.3.0-26-generic.

The syncing starts right after the error message. Unfortunately it finishs after 12-13hours of sending in my case but I'm getting this error again and it won't sync those other snapshots I did before.

Anyone an idea how to fix this?

**Comment by nicedevil007** [comment]
Did a complete fresh run of this sync now with `--debug` switch

There was an errror at the start:

```
root@ubuntu:~# syncoid --no-sync-snap --sendoptions="--raw -c -L" nas/docker root@192.168.1.250:backup/docker --debug > debug.out
bad property list: invalid property 'A-Hpd'
usage:
        get [-rHp] [-d max] [-o "all" | field[,...]]
            [-t type[,...]] [-s source[,...]]
            <"all" | property[,...]> [filesystem|volume|snapshot|bookmark] ...

The following properties are supported:
......

Sizes are specified in bytes with standard units such as K, M, G, etc.

User-defined properties can be specified by using a name containing a colon (:).

The {user|group|project}[obj]{used|quota}@ properties must be appended with
a user|group|project specifier of one of these forms:
    POSIX name      (eg: "matt")
    POSIX id        (eg: "126829")
    SMB name@domain (eg: "matt@sun")
    SMB SID         (eg: "S-1-234-567-89")
snapshot listing failed, trying fallback command at /usr/sbin/syncoid line 1550.

```

**Comment by phreaker0** [comment]
you are using the master version apparently, there is debug typo which i let slip in, see #480 and pr to fix it here #481 
But the point of this debug typo was to force the usage of the fallback code which should work like the other one. You are saying the first replication worked but the following ones aren't sending new snapshots?

Can you provide some more information, especially which snapshots aren't replicated and the snapshot for the two datasets:

zfs list -t snapshot -d 1 nas/docker
ssh root@192.168.1.250 zfs list -t snapshot -d 1 backup/docker

**Comment by nicedevil007** [comment]
thx @phreaker0 

just did everything as described in the install.md file to install it on ubuntu :)
So git clone ... master ;)
Now I downloaded the release of 2.0.3 and everything works as intended, no error anymore!
